src/runner/tests/driver-test-runner.js

```diff
@@ -0,0 +1,54 @@
+import test from 'ava';
+
+import {
+  NVDASettingResponses,
+  VOSettingResponses,
```

Comment:
Because the function's implementation is implementation-agnostic, a lot of these tests are essentially asserting the constants. I'd be comfortable with a smaller suite of tests that focused on the functionality of the utility using test values as input.